edk2toollib/uefi/edk2/parsers/inf_parser.py

```diff
@@ -12,7 +12,7 @@
 from edk2toollib.uefi.edk2.parsers.base_parser import HashFileParser
 
 AllPhases = ["SEC", "PEIM", "PEI_CORE", "DXE_DRIVER", "DXE_CORE", "DXE_RUNTIME_DRIVER", "UEFI_DRIVER",
-             "SMM_CORE", "DXE_SMM_DRIVER", "UEFI_APPLICATION"]
+             "SMM_CORE", "DXE_SMM_DRIVER", "UEFI_APPLICATION", "MM_STANDALONE", "MM_CORE_STANDALONE"]
```

Comment:
Should we include HOST_APPLICATION?

Reply:
Absolutely. I'll add it in. This list was fairly outdated so if there are more then just HOST_APPLICATION, let me know.

Reply:
Added. Thanks for the review!